ledger/ledger-api-tests/suites/src/main/scala/com/daml/ledger/api/testtool/suites/v1_8/ActiveContractsServiceIT.scala

```diff
@@ -398,11 +398,14 @@ class ActiveContractsServiceIT extends LedgerTestSuite {
     for {
       _ <- ledger.create(alice, WithObservers(alice, Seq(alice, bob)))
       contracts <- ledger.activeContracts(alice)
-      Seq(ce) = contracts
-    } yield assert(
-      ce.observers == Seq(bob),
-      s"Expected observers to only contain $bob, but received ${ce.observers}",
-    )
+    } yield {
+      contracts.foreach(ce =>
```

Comment:
the logic here has changed, if you get empty `contracts` you wont get an error anymore. you could add assert that `contract` is nonEmpty

Reply:
or getting `.head`

Reply:
:+1: